proposals/field-keyword-nullability.md

```diff
@@ -87,3 +98,41 @@ At each explicit or implicit 'return' in the setter, a warning is reported if th
 This formulation is intentionally very similar to ordinary fields in constructors. Essentially, because only the property accessors can actually refer to the backing field, the setter is treated as a "mini-constructor" for the backing field.
 
 Much like with ordinary fields, we usually know the property was initialized in the constructor because it was set, but not necessarily. Simply returning within a branch where `Prop != null` was true is also good enough for our constructor analysis, since we understand that untracked mechanisms may have been used to set the property.
+
+## Alternatives
+
+In addition to the *null-resilience* approach outlined above, the working group suggests the following alternatives for the LDM's consideration:
+
+### Do nothing
+
+We could introduce no special behavior at all here. In effect:
+- Treat a field-backed property the same way auto-properties are treated today--must be initialized in constructor except when marked required, etc.
+- No special treatment of the field variable when analyzing property accessors. It is simply a variable with the same type and nullability as the property.
+
+Note that this would result in nuisance warnings for "lazy property" scenarios, in which case users would likely need to assign `null!` or similar to silence constructor warnings.  
+A "sub-alternative" we can consider is to also completely ignore properties using `field` keyword for nullable constructor analysis. In that case, there would be no warnings anywhere about the user needing to initialize anything, but also no nuisance for the user, regardless of what initialization pattern they may be using.
```

Comment:
(A scary sub-alternative, from my perspective 😄)

Reply:
I should probably make clear that "do nothing" could be a temporary state, and we could pursue a permanent with better null safety in the future.